signatures/alert_signatures.py

```diff
@@ -0,0 +1,1012 @@
+    "ET CURRENT_EVENTS Possible ZyXELs ZynOS Configuration Download Attempt (Contains Passwords)": MicroAttackStage.DATA_EXFILTRATION,
+    "ET CURRENT_EVENTS QNAP Shellshock CVE-2014-6271": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET CURRENT_EVENTS Terse alphanumeric executable downloader high likelihood of being hostile": MicroAttackStage.DATA_DELIVERY,
+    "ET DNS Query for .su TLD (Soviet Union) Often Malware Related": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET DNS Query to a .tk domain - Likely Hostile": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET DOS Microsoft Remote Desktop (RDP) Syn then Reset 30 Second DoS Attempt": MicroAttackStage.NETWORK_DOS,
+    "ET DOS Possible NTP DDoS Inbound Frequent Un-Authed MON_LIST Requests IMPL 0x03": MicroAttackStage.NETWORK_DOS,
+    "ET DOS Possible SSDP Amplification Scan in Progress": MicroAttackStage.NETWORK_DOS,
+    "ET EXPLOIT Possible GoldenPac Priv Esc in-use": MicroAttackStage.USER_PRIV_ESC,
+    "ET EXPLOIT Possible Postfix CVE-2014-6271 attempt": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET EXPLOIT Possible Pure-FTPd CVE-2014-6271 attempt": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET EXPLOIT Possible SpamAssassin Milter Plugin Remote Arbitrary Command Injection Attempt": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET EXPLOIT REDIS Attempted SSH Key Upload": MicroAttackStage.USER_PRIV_ESC,
+    "ET FTP Suspicious Quotation Mark Usage in FTP Username": MicroAttackStage.ACCT_MANIP,
+    "ET INFO Executable Download from dotted-quad Host": MicroAttackStage.COMMAND_AND_CONTROL,  # Do we need a malware injection stage?
+    "ET INFO Possible Windows executable sent when remote host claims to send a Text File": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET INFO WinHttp AutoProxy Request wpad.dat Possible BadTunnel": MicroAttackStage.DATA_EXFILTRATION,  # This is more "man in the middle"
+    "ET MOBILE_MALWARE Android/Code4hk.A Checkin": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET P2P TOR 1.0 Outbound Circuit Traffic": MicroAttackStage.DEFENSE_EVASION,
+    "ET POLICY DNS Update From External net": MicroAttackStage.DATA_MANIPULATION,
+    "ET POLICY Executable and linking format (ELF) file download": MicroAttackStage.COMMAND_AND_CONTROL,  # Will need to be checked out
+    "ET POLICY Executable and linking format (ELF) file download Over HTTP": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET POLICY Http Client Body contains pass= in cleartext": MicroAttackStage.PRIV_ESC,  # Once again, check this out.
+    "ET POLICY Incoming Basic Auth Base64 HTTP Password detected unencrypted": MicroAttackStage.USER_PRIV_ESC,
+    "ET POLICY MS Remote Desktop Administrator Login Request": MicroAttackStage.ROOT_PRIV_ESC,
+    "ET POLICY MS Terminal Server Root login": MicroAttackStage.ROOT_PRIV_ESC,
+    "ET POLICY Outgoing Basic Auth Base64 HTTP Password detected unencrypted": MicroAttackStage.USER_PRIV_ESC,  # Outgoing vs. incoming?
```

Comment:
Should comments like this just remain as they are?

Reply:
Yes